### ISSUE TITLE
WCAG 2.2 1.4.6 Contrast (Enhanced) failure in WCAG 2.2 draft

### ISSUE THREAD

**Issue by w3cbot** [w3c/a11y-review#160]
**This is a tracker issue.** Only discuss things here if they are a11y group internal meta-discussions about the issue. **Contribute to the actual discussion at the following link:**

§ https://github.com/w3c/wcag/issues/3105
